index.html

```diff
@@ -1,53 +1,124 @@
-        When you're done, Dafny can <strong>compile your code to C#, Java, JavaScript or Go</strong> (more to come!), so it can integrate with your existing workflow.
-    </p>
-    <p>
-        Dafny will give you <strong>assurance that your code meets the specifications you write</strong>, while letting you write both code and specifications in the Dafny programming language itself.
-        Since verification is an integral part of development, it will thus <strong>reduce the risk of costly late-stage bugs</strong> that are typically missed by testing.
-    </p>
-    <p>
-        Dafny has support for common programming concepts such as <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-class-types">classes and trait inheritance</a>,
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#181-inductive-datatypes">inductive datatypes</a> that can have methods and are suitable for pattern matching,
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#182-co-inductive-datatypes">lazily unbounded datatypes</a>,
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-subset-types">subset types e.g. for bounded integers</a>,
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-lambda-expressions">lambdas</a>,
-        and <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-collection-types">immutable and mutable data structures</a>.
-    </p>
-    <p>
-        Dafny also offers an extensive toolbox for mathematical proofs,
-        such as <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#sec-forall-statement">unbounded and bounded quantifiers</a>,
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#1923-calc-statement">calculational proofs</a>,
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#51-specification-clauses">pre- and post-conditions,
-        termination conditions, loop invariants, and read/write specifications</a>.
-    </p>
-
-    <h2>Get Started</h2>
-    <p>
-        Try it out by installing the Visual Studio Code <a href="https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode">extension</a>, which installs all dependencies apart from .NET 6.
-        To get a better sense of Dafny's capabilities, follow the <a href="https://dafny.org/dafny/OnlineTutorial/guide">tutorial</a>.
-    </p>
-
-    <h2>Get Involved</h2>
-    <p>
-        Development happens on <a href="https://github.com/dafny-lang/dafny">GitHub</a>.
-    </p>
+
+<div>
+<h1>Dafny</h1>
+</div>
+
+<table style="width:100%">
+<tr>
+<td style="width:15%">
+<img src="Banner/dafny-logo.jpg" alt="Dafny" width="180" height="150">
+</td>
+<td style="width:45%">
+<p>
+<b>Dafny</b> is a verification-aware programming language that has native support for recording specifications and is equipped with a static program verifier. By blending sophisticated automated reasoning with familiar programming idioms and tools, Dafny empowers developers to write provably correct code (w.r.t. specifications),
```

Comment:
Could you add a few more line-breaks to the longer lines? I'm a fan of https://sembr.org/ as a general guideline but don't like to be too strict about it.

Reply:
Done